daemon/daemon.go

```diff
@@ -1073,6 +1073,15 @@ func NewDaemon() (*Daemon, error) {
 			log.Info("k8s mode: Allowing localhost to reach local endpoints")
```

Comment:
Typo in commit message:

"In either of the above scenarios, traffic via NodePort was automatically
allowed if the traffic was SNATed Cilium" --> "In either of the above scenarios, traffic via NodePort was automatically allowed if the traffic was SNATed. Cilium..."

Reply:
Fixed.